Software/src/GrabConfigWidget.cpp

```diff
@@ -59,7 +60,7 @@ GrabConfigWidget::~GrabConfigWidget()
 
 void GrabConfigWidget::showConfigFor(QRect widgetGeometry, int buttonCenter)
 {
-	QRect screen = QApplication::desktop()->screenGeometry(widgetGeometry.center());
+	QRect screen = QGuiApplication::screenAt(widgetGeometry.center())->geometry();
```

Comment:
This was introduced in 5.10. It breaks build on 16.04 LTS (at least on my CI VM) as far as I can tell and the old version doesn't seem to be deprecated?

Reply:
Oh hi :)
[It is deprecated](https://doc.qt.io/qt-5/qdesktopwidget-obsolete.html#screenGeometry-2) since 5.11, the [one that isn't (yet?)](https://doc.qt.io/qt-5/qdesktopwidget.html#screenGeometry-1) takes a `QWidget*`.

I see the linux CI still uses Qt 5.5, which is 5 years and almost 4 LTS releases old, maybe it can be bumped? (and Qt6 is planned for [the end of the year](https://www.qt.io/blog/2019/08/07/technical-vision-qt-6))

Reply:
Hm. 16.04 LTS is supported until Apr 2021, but seems stuck at Qt 5.5. Even upgrading to 18.04 appears to yield 5.9, which wouldn't be enough and I'd rather stay on an LTS version for the CI.

Reply:
maybe there's a ppa for 5.12?
if not, I'm fine with something like this
```cpp
	#if (QT_VERSION >= QT_VERSION_CHECK(5, 10, 0))
	QRect screen = QGuiApplication::screenAt(widgetGeometry.center())->geometry();
	#else
	QApplication::desktop()->screenGeometry(widgetGeometry.center());
	#endif
```

Reply:
I'd like to remain compatible with the major Ubuntu LTS release. I'll upgrade the VM to 18.04 but since that's probably not going to fix the issue please add the version switch. Thank you :)

Reply:
Done